activiti-cloud-services-runtime-bundle/activiti-cloud-services-connectors/src/main/java/org/activiti/services/connectors/model/IntegrationResultEvent.java

```diff
@@ -41,6 +43,13 @@ public IntegrationResultEvent(String id,
         this.variables = variables;
     }
 
+    public IntegrationResultEvent(String executionId,
```

Comment:
Same here. Probably we don't need this constructor.

Reply:
+1